dll/cpl/usrmgr/userprops.c

```diff
@@ -476,7 +476,7 @@ AddSelectedGroupsToUser(HWND hwndDlg,
                                                  (LPBYTE)&memberInfo, 1);
                 if (status == NERR_Success)
                 {
-                    DebugPrintf(_TEXT("Selected group: %s"), szGroupName);
+                    TRACE("Selected group: %s", szGroupName);
```

Comment:
Mixing CHAR and TCHAR

Reply:
Would you recommend wcstombs() here ?

Reply:
`szGroupName` can be CHAR or WCHAR, depending on the compiler settings (because it's TCHAR).
That is a very obsolete choice, but sadly some code still uses TCHAR.
So either rewrite the entire thing to use WCHAR, or take into account the current setting and conditionally (compile time!) use the correct format specifier here.

Reply:
Would you agree that the most desirable option is to work on refactoring toward a clean WCHAR implementation for the sake of value for the project ?

Reply:
To be left to the maintainer of usrmgr (Eric).

Reply:
To be managed by the maintainer of usrmgr (Eric).

Reply:
Thanks @HBelusca for the finalization.
OK. So your change is now making the PR OK isn't it ?

Reply:
@learn-more : Do you approve it to, are are still changes required ?